t/ui/18-tests-details.t

```diff
@@ -446,6 +448,8 @@ my $post
 $get         = $t->get_ok($baseurl . 'tests/99963')->status_is(200);
 @worker_text = $get->tx->res->dom->find('#assigned-worker')->map('all_text')->each;
 like($worker_text[0], qr/[ \n]*Assigned worker:[ \n]*localhost:1[ \n]*/, 'worker still displayed when job set to done');
+@group_description_text = $get->tx->res->dom->find('#group-description')->map('all_text')->each;
+like($group_description_text[0], qr/[ \n]*##Test description\s+with bugref bsc#1234[ \n]*/, 'test group description is missing');
```

Comment:
Hm, why this repetition?

Reply:
I tried to do the same way as test for assigned-worker = let's say we check  it second time to be really sure that it isn't gone after refresh. Or I can remove it as well